x-pack/test_serverless/api_integration/test_suites/common/core/compression.ts

```diff
@@ -7,35 +7,47 @@
 
 import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../../ftr_provider_context';
+import { RoleCredentials } from '../../../../shared/services';
 
 export default function ({ getService }: FtrProviderContext) {
-  const supertest = getService('supertest');
   const svlCommonApi = getService('svlCommonApi');
+  const svlUserManager = getService('svlUserManager');
+  let roleAuthc: RoleCredentials;
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
 
   const compressionSuite = (url: string) => {
     it(`uses compression when there isn't a referer`, async () => {
-      await supertest
+      await supertestWithoutAuth
         .get(url)
         .set('accept-encoding', 'gzip')
         .set(svlCommonApi.getInternalRequestHeader())
+        .set(roleAuthc.apiKeyHeader)
         .then((response) => {
           expect(response.header).to.have.property('content-encoding', 'gzip');
         });
     });
 
     it(`uses compression when there is a whitelisted referer`, async () => {
-      await supertest
+      await supertestWithoutAuth
         .get(url)
         .set('accept-encoding', 'gzip')
         .set(svlCommonApi.getInternalRequestHeader())
         .set('referer', 'https://some-host.com')
+        .set(roleAuthc.apiKeyHeader)
         .then((response) => {
           expect(response.header).to.have.property('content-encoding', 'gzip');
         });
     });
   };
 
   describe('compression', () => {
+    before(async () => {
+      // TODO-TRE: Do we really need admin for checking referrers?
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
```

Comment:
_suggestion_ : please confirm it with code owners before merge

Reply:
@TinaHeiligers this seems like something we should not need admin for. 
Is there a reason you know of?

Reply:
Dropped the comment: https://github.com/elastic/kibana/pull/181741/commits/56efd6fc6febd1275720348a6c5c0b5b940be086